Assets/MixedRealityToolkit/_Core/Utilities/Async/MixedRealityToolkit.Core.Utilities.Async.asmdef

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "Microsoft.MixedReality.Toolkit.Core.Utilities.Async",
```

Comment:
why does the toolkit need a new assembly? all major components reference this new one...

do we get a benefit above having this class in the core assembly?

Reply:
Yes, it's one less thing that has to recompile, and adds a bit of security to the async utilities, as there were a few public methods and classes that should have been internal when I initially wrote it.